ASWF/meetings/working_groups/configs/2020-11-03.md

```diff
@@ -0,0 +1,147 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+November 3, 2020
+
+Host: Michael Dolan
+
+Attendees:
+  * [X] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Kevin Wheatley (_TSC_) - Framestore
+  * [X] Dennis Adams - Sony
+  * [X] Alessandra Tomassi - Industrial Light & Magic
+  * [X] J Schulte - Industrial Light & Magic
+
+Apologies:
+  * Carol Payne
+
+# **OCIO Configs Working Group Meeting Notes**
+
+* Time change:
+    - Michael: All daylight savings shifts are now done and Thomas is unable to 
+      attend. Can we shift one hour earlier next time?
+    - All ok with this. We will try to not change time mid-daylight savings 
```

Comment:
Thanks, as discussed with Michael yesterday, it seems like the most sensible thing to do, i.e. wait for the month and half of daylight savings dust to settle.